frontend/src/app/actions/newsletter-form.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from 'redux-actions';
+
+const actions = createActions(
+  {
+    newsletterFormSetEmail: email => email,
+    newsletterFormSetPrivacy: privacy => privacy
+  },
+  'newsletterFormSetFailed',
+  'newsletterFormSetSubmitting',
+  'newsletterFormSetSucceeded'
+);
+
+const subscribeActions = createActions({
+  newsletterFormSubscribe: (dispatch, email) => {
```

Comment:
Eh, yeah, I think it just rubs me the wrong way for reasons I'm unable to really explain. But, this looks like a pretty textbook example from the docs. So, may just be I need to soak my head in more docs